<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="solveButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>470</y>
      <width>129</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Решить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>530</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Z(X, Y) = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>310</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>390</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Nx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>390</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ny</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>60</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>От:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>До:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>60</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Шаг:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>210</y>
      <width>231</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Задать таблицу</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xFromEdit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yFromEdit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xToEdit">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="yToEdit">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xStepEdit">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yStepEdit">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xFindEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yFindEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nxEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>430</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nyEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>430</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
